### PR TITLE
Update gdpr-dsr-Office365.md

### PR DESCRIPTION
**Pull request by msbemba**
Added yammer content search to the doc

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2571

File: microsoft-365/compliance/gdpr-dsr-Office365.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ A quick way to identify the Office 365 applications that people in your organiza
...
...a your organization creates and stores using in Office 365, you may want to first consider which applications people have most likely used to author the data you're looking for. Microsoft estimates that over 90% of an organization's data that is stored in Office 365 is authored in Word, Excel, PowerPoint, OneNote, and Outlook. Documents authored in these Office applications, even if purchased through Microsoft 365 Apps for enterprise or an Office perpetual license, are most likely stored on a SharePoint Online site, in a user's OneDrive for Business account, or in a user's Exchange Online mailbox. That means you can use the Content Search eDiscovery tool to search (and perform other DSR-related actions) across SharePoint Online sites, OneDrive for Business accounts, and Exchange Online mailboxes (including the sites and mailboxes associated with Microsoft 365 Groups, Microsoft Teams, EDU Assignments) to find documents and mailbox items that may be relevant to the DSR you're investigating. You can also use the Content Search tool to discover Customer Data authored in other Office 365 applications. You can now use content search for Yammer content if Yammer is running in native mode.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-dsr-Office365.md:92]:
```suggestion
When looking for personal data within the larger set of data your organization creates and stores in Office 365, you may want to first consider which applications people have most likely used to author the data you're looking for. Microsoft estimates that over 90% of an organization's data that is stored in Office 365 is authored in Word, Excel, PowerPoint, OneNote, and Outlook. Documents authored in these Office applications, even if purchased through Microsoft 365 Apps for enterprise or an Office perpetual license, are most likely stored on a SharePoint Online site, in a user's OneDrive account, or in a user's Exchange Online mailbox. That means you can use the Content Search eDiscovery tool to search (and perform other DSR-related actions) across SharePoint Online sites, OneDrive accounts, and Exchange Online mailboxes (including the sites and mailboxes associated with Microsoft 365 Groups, Microsoft Teams, EDU Assignments) to find documents and mailbox items that may be relevant to the DSR you're investigating. You can also use the Content Search tool to discover Customer Data authored in other Office 365 applications. Moreover, you can now use the Content Search tool for Yammer content if Yammer is running in native mode.
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-dsr-Office365.md]:
```suggestion
When looking for personal data within the larger set of data your organization creates and stores in Office 365, you may want to first consider which applications people have most likely used to author the data you're looking for. Microsoft estimates that over 90% of an organization's data that is stored in Office 365 is authored in Word, Excel, PowerPoint, OneNote, and Outlook. Documents authored in these Office applications, even if purchased through Microsoft 365 Apps for enterprise or an Office perpetual license, are most likely stored on a SharePoint Online site, in a user's OneDrive for Business account, or in a user's Exchange Online mailbox. That means you can use the Content Search eDiscovery tool to search (and perform other DSR-related actions) across SharePoint Online sites, OneDrive for Business accounts, and Exchange Online mailboxes (including the sites and mailboxes associated with Microsoft 365 Groups, Microsoft Teams, EDU Assignments) to find documents and mailbox items that may be relevant to the DSR you're investigating. You can also use the Content Search tool to discover Customer Data authored in other Office 365 applications. Moreover, you can now use the Content Search tool for Yammer content if Yammer is running in native mode.
```

Comment (JohanFreelancer9) [reply microsoft-365/compliance/gdpr-dsr-Office365.md]: Please remove the word "using" from the first line. Also, change the last line as indicated.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@robmazz Gentle reminder, please merge this PR. Thanks

**Comment by robmazz** [comment]
@msbemba - thanks for the feedback. All updates to GDPR content must be approved by MS CELA and the frontline CELA reps for each workload. Closing this PR, but will forward recommendation to CELA contacts for review.

**Pull request closed without merging**